axlearn/common/attention.py

```diff
@@ -3589,6 +3590,30 @@ def _forward_for_mode(
 
         return all_layer_states, self._aggregate_layer_outputs(all_layer_outputs)
 
+    # pylint: disable=unused-argument
```

Comment:
Could we use `del all_layer_outputs` in function body or `disable-next`?

Reply:
Done